### PR TITLE
Lenses do not need to be `implicit`

### PR DESCRIPTION
**Pull request by armanbilge**

### CODE REVIEW

**Review by rpiaggio — APPROVED** [review]
LGTM

File: gen/rules/src/main/scala/clue/gen/Generator.scala [hunk 1]
```diff
@@ -236,7 +236,7 @@ trait Generator {
           val lensesDef = params.map { param =>
             val thisType  = qualifiedNestedType(nestPath, Type.Name(camelName))
             val childType = param.typeTree(nextPath, nextTypes)
-            q"implicit val ${Pat.Var(Term.Name(param.name))}:  monocle.Lens[$thisType, $childType] = monocle.macros.GenLens[$thisType](_.${Term
+            q"val ${Pat.Var(Term.Name(param.name))}:  monocle.Lens[$thisType, $childType] = monocle.macros.GenLens[$thisType](_.${Term
```
Comment (cquiroz) [inline gen/rules/src/main/scala/clue/gen/Generator.scala:239]: Would it help on js size to declar them `lazy‘?

Comment (armanbilge) [reply gen/rules/src/main/scala/clue/gen/Generator.scala:239]: Probably not after https://github.com/scala-js/scala-js/pull/4681? I don't know.

Comment (cquiroz) [reply gen/rules/src/main/scala/clue/gen/Generator.scala:239]: Oh that's nice

**Pull request merged**